examples/data/geodataset_catalog.yml

```diff
@@ -1,4 +1,6 @@
 ---
+meta:
+  root: examples/data
```

Comment:
so this is the old way of registering a root, now the format should be: 
```yml
meta:
    roots:
        - examples/data
```

I'm not sure we ever discussed whether we want to deprecate the old format or not. What do you think?

Reply:
These are still in the current data catalogs. We can decide to deprecate in another PR